doc/man3/SSL_CTX_set_record_padding_callback.pod

```diff
@@ -46,6 +50,10 @@ SSL_CTX_set_block_padding() and SSL_set_block_padding() pads the record to a mul
 of the B<block_size>. A B<block_size> of 0 or 1 disables block padding. The limit of
 B<block_size> is SSL3_RT_MAX_PLAIN_LENGTH.
 
+SSL_CTX_set_block_padding_ex() and SSL_set_block_padding_ex() do similarly but
+allow the caller to separately specify the padding block size to be applied to
+handshake and application data messages.
```

Comment:
What happens to alert records in this case?

Reply:
Good question: Checking that now...

Reply:
So, yes, [RFC8446, section 5.4](https://datatracker.ietf.org/doc/html/rfc8446#section-5.4) says "Padding for encrypted Handshake or Alert records must still be handled at the TLS layer, though." I'll change so as to apply `hs_padding` to alerts as well if appropriate.